src/elements/content-uploader/ContentUploader.js

```diff
@@ -131,6 +132,7 @@ class ContentUploader extends Component<Props, State> {
         onMinimize: noop,
         onCancel: noop,
         isFolderUploadEnabled: false,
+        isResumableUploadsEnabled: true,
```

Comment:
Is this meant to be enabled by default? If so, is it needed?

Reply:
It's set to be enabled by default right now for testing purposes, but it doesn't necessarily have to be enabled by default.